.gitignore

```diff
@@ -1,2 +1,3 @@
 .vscode
-.mypy_cache
\ No newline at end of file
+.mypy_cache
+.ipynb_checkpoints/
```

Comment:
I don't think there are .ipynb files in the repo?

Reply:
Good question. When I was editting the README in Jupyter, there was a file called .ipynb_checkpoints/ that was automatically created with this file stored in it:

![image](https://user-images.githubusercontent.com/35017006/106499748-d1dffa00-6475-11eb-9401-b2bc50e37f0e.png)

I think Jupyter creates these automatically. I tried to delete it and it was created again so just thought I might add this in the gitignore. Let me know if this makes sense and if this solution is ok!

Reply:
Yeah, those files are generated by the notebooks. I think it is fine as long as you don't add them when do commit.